sonar-project.properties

```diff
@@ -0,0 +1,13 @@
+###############################################################################
```

Comment:
Do we require sonar? There's no go code to test or scan in this project.

Reply:
We dont nee sonar.

Reply:
Removed the sonar file.